.github/workflows/ci.yaml

```diff
@@ -123,68 +119,22 @@ jobs:
           ruby: true
           gcp-provider: ${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}
           gcp-service-account: ${{ secrets.GCP_SERVICE_ACCOUNT }}
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v4
-        with:
-          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
-          aws-region: us-east-1
-      - name: Fetch AWS secrets
-        uses: aws-actions/aws-secretsmanager-get-secrets@v2
-        with:
-          secret-ids: |
-            mobile-app-android-upload-key-passphrase
-      - name: Load code signing key
-        run: |
-          cd androidApp
-          aws secretsmanager get-secret-value --secret-id mobile-app-android-upload-key --output json | jq -r '.SecretBinary' | base64 --decode > upload-keystore.jks
       - name: Build Android app
         env:
           FIREBASE_KEY: ${{ secrets.FIREBASE_KEY }}
           GOOGLE_APP_ID_ANDROID: ${{ secrets.GOOGLE_APP_ID_ANDROID_STAGING }}
-          KEYSTORE_FILE: "${{ github.workspace }}/androidApp/upload-keystore.jks"
-          KEYSTORE_PASSWORD: ${{ env.MOBILE_APP_ANDROID_UPLOAD_KEY_PASSPHRASE }}
-          KEY_ALIAS: "upload"
-          KEY_PASSWORD: ${{ env.MOBILE_APP_ANDROID_UPLOAD_KEY_PASSPHRASE }}
           MAPBOX_SECRET_TOKEN: ${{ secrets.MAPBOX_SECRET_TOKEN }}
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
           SENTRY_DSN_ANDROID: ${{ secrets.SENTRY_DSN_ANDROID }}
           SENTRY_ENVIRONMENT: "staging"
 
         run: |
-          bundle exec fastlane android build flavor:Staging
+          bundle exec fastlane android build flavor:Staging build_type:Debug
       - uses: actions/upload-artifact@v4
         with:
           name: android-apk
-          path: androidApp/build/outputs/apk/staging/release/androidApp-staging-release.apk
+          path: androidApp/build/outputs/apk/staging/debug/androidApp-staging-debug.apk
       - uses: actions/upload-artifact@v4
         with:
           name: android-aab
```

Comment:
Uploading the aab is only useful for deploying, and uploading the apk is only useful for giving our non-Google-Play-authenticated beta testers a build to sideload. Neither of these is a consideration in these debug builds, so we should probably skip the uploads.